assets/js/components/HostDetails/HostDetailsPage.jsx

```diff
@@ -51,7 +51,6 @@ function HostDetailsPage() {
   if (!host) {
     return <div>Not Found</div>;
   }
-
```

Comment:
mmm, let's better revert this strange change, how was done by me most probably :sweat_smile: